DataProcessor/DataUnpacker.h

```diff
@@ -42,17 +42,25 @@ class DataUnpacker : public QObject
     Q_PROPERTY(bool door MEMBER door NOTIFY dataChanged);
     Q_PROPERTY(bool mcu_check MEMBER mcu_check NOTIFY dataChanged);
     Q_PROPERTY(bool imd_status MEMBER imd_status NOTIFY dataChanged);
-    Q_PROPERTY(bool mps_enable MEMBER mps_enable NOTIFY dataChanged);
+    Q_PROPERTY(bool discharge_enable MEMBER discharge_enable NOTIFY dataChanged);
     Q_PROPERTY(bool mppt_contactor MEMBER mppt_contactor NOTIFY dataChanged);
     Q_PROPERTY(bool motor_controller_contactor MEMBER motor_controller_contactor NOTIFY dataChanged);
```

Comment:
```suggestion
```
Since we aren't displaying these on the driver dashboard, they should be removed